src/compiler/scala/tools/nsc/typechecker/AnalyzerPlugins.scala

```diff
@@ -349,6 +369,18 @@ trait AnalyzerPlugins { self: Analyzer =>
     def accumulate = (tpe, p) => p.pluginsTypedReturn(tpe, typer, tree, pt)
   })
 
+  /** @see AnalyzerPlugin.pluginsImplicitSearch */
+  def pluginsImplicitSearch(search: ImplicitSearch): ImplicitSearch = invoke(new CumulativeOp[Unit] {
```

Comment:
Should have return type `Unit` (also below)

Reply:
Yes, I pushed a bad commit.